### PR TITLE
Initialize the project

### PR DESCRIPTION
**Pull request by lubo**

### CODE REVIEW

File: .github/dependabot.yml
Comment (mitas1) [inline .github/dependabot.yml:10]: This seems like too often, but I expect you are going to do maintenance, so pick whatever works for you ;)

File: .eslintrc.js
Comment (mitas1) [inline .eslintrc.js:25]:
```suggestion
  // Local rules have to be prefixed with `rulesdir/`. Let's remove the preset
```

**Review by mitas1 — DISMISSED** [review]
Thank you @lubo! It looks good to me.

File: .github/dependabot.yml
Comment (lubo) [reply .github/dependabot.yml:10]: Absolutely not too often, this means a whole day can go by without a fix. I think this is just and correct, I'll be doing the maintenance.

**Review by mitas1 — APPROVED** [review]

**Comment by lubo** [comment]
I've cleaned up the history and I'll merge after you approve one more time.

**Review by mitas1 — APPROVED** [review]

**Pull request merged**
